api/src/bigquerytest/java/org/pmiops/workbench/testconfig/TestJpaConfig.java

```diff
@@ -0,0 +1,60 @@
+package org.pmiops.workbench.testconfig;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.persistence.EntityManagerFactory;
+import javax.sql.DataSource;
+import java.util.Properties;
+
+@Configuration
+@EnableJpaRepositories(basePackages = { "org.pmiops.workbench.cdr" })
+@EnableTransactionManagement
+public class TestJpaConfig {
```

Comment:
Why didn't we need this before? (Is this the first time we've been reading from the CDR DB in tests?)

Reply:
We didn't need this before because the ParticipantCounter in the CohortMaterializationService didn't have a DomainLookupService(this is what i moved into the ParticipantCounter). The DomainLookupService has a CriteriaDao(This is why we need TestJpaConfig). So this test has to connect to BigQuery and the cdr database. Before this PR it only connected to BigQuery. Hope that makes sense.